src/clib/lib/job_queue/queue_driver.cpp

```diff
@@ -85,6 +85,11 @@ static bool queue_driver_set_generic_option__(queue_driver_type *driver,
         if (strcmp(MAX_RUNNING, option_key) == 0) {
             int max_running_int = 0;
             if (util_sscanf_int(value, &max_running_int)) {
+
+                if (max_running_int < 0) {
```

Comment:
This should have no effect when the validation is done in res_config (optionally in queue_config).
Should remove?

Reply:
max_running_int = 0 should mean unlimited so something seems off.

Reply:
Seems a bit off as max_running = 1 is generally not we want, but rather max_running=0 (unlimited).

Reply:
I have no objections to this. Question is if this should be removed altogether, since the validation should in theory make sure this will never go into effect.

Reply:
Yea, I think remove all together is the best. Adding sanitation to validation just complicates things.